src/lib/formats/img_dsk.cpp

```diff
@@ -0,0 +1,214 @@
+// license:BSD-3-Clause
+// copyright-holders: F. Ulivi
+/*********************************************************************
+
+    img_dsk.cpp
+
+    "IMG" disk format
+
+    This format is just a raw image of every sector on SSDD 8" floppy
+    disks as used in Intel MDS-II systems.
+    Files with this format have no header/trailer and are exactly
+	512512 bytes in size (52 sectors, 1 head, 77 tracks,
+    128 bytes per sector).
+
+*********************************************************************/
+
+#include "emu.h"
+#include "img_dsk.h"
+
+// Debugging
+#define VERBOSE 0
+#define LOG(...)  do { if (VERBOSE) printf(__VA_ARGS__); } while (false)
```

Comment:
You should use one of the `osd_printf_*` things rather than `printf` here to get consistent results across MinGW, Linux, macOS, and MSVCRT.